bom/application/pom.xml

```diff
@@ -3593,6 +3593,11 @@
                 <artifactId>hamcrest</artifactId>
                 <version>${hamcrest.version}</version>
             </dependency>
+            <dependency>
+                <groupId>me.escoffier.certs</groupId>
+                <artifactId>certificate-generator-junit5</artifactId>
+                <version>0.3.0</version>
+            </dependency>
```

Comment:
@cescoffier I don't think this should be in the BOM. I would move it to the `build-parent`.

Reply:
Ok, I will do that in my next PR about kafka TLS support (which as I needed to fix something in the junit5 extension)